### ISSUE TITLE
token autoscan does not work

### ISSUE THREAD

**Issue by RaZeRiel** [ethereum/mist#4087]
like the title says, token autoscan does not work and i don't why.
i have already deleted all folders & reinstalled the wallet, tried 0.10.0 version and still not working.

the log says:
[2018-09-04T19:56:55.728] [INFO] (ui: browser) - Error parsing token list: SyntaxError: Unexpected token : in JSON at position 3
[2018-09-04T19:56:55.729] [WARN] (ui: browser) - %cUnhandled rejection Error parsing token list: SyntaxError: Unexpected token : in JSON at position 3 color: red

```
Version: `0.11.1`
OS & Version: windows
Node version: `geth 1.8.13`
Number of blocks synchronized: 6 271 616
```

![git](https://user-images.githubusercontent.com/42977898/45048516-6e6ef980-b07c-11e8-81c2-6f11d059929f.JPG)

![git2](https://user-images.githubusercontent.com/42977898/45048838-30bea080-b07d-11e8-8719-7cb3cf4ede27.JPG)

[all.log.zip](https://github.com/ethereum/mist/files/2349684/all.log.zip)

**Comment by evertonfraga** [comment]
Thanks for reporting @RaZeRiel, it will be sorted out soon!

**Comment by sycomix** [comment]
this bug also exists in Linux 
![image](https://user-images.githubusercontent.com/6117352/46242851-1da8a300-c393-11e8-977f-08d69b3b4b3e.png)

**Comment by mpge** [comment]
Confirming bug also exists in MacOS

**Comment by psy0rz** [comment]
confirmed, it seems this url that it is trying to download is invalid:

https://raw.githubusercontent.com/MyEtherWallet/ethereum-lists/master/tokens/tokens-eth.json

**Comment by evertonfraga** [comment]
Yeah, the source changed URLs. That was fixed in our local repository and will be available as soon as we release 0.12.0. 

Thanks for reporting.

**Comment by psy0rz** [comment]
Shouldn't we be our "own" source? e.g. fork that repository and create a cronjob somewhere that updates it automatically. 

This way we can fix it retroactively when they change something, instead of having to wait on the next release.

Also: Should our json parser handle data from an "untrusted" source? What if someone crafts a special file that exploits a weakness in our json handling code?

**Comment by UGatgithub** [comment]
I can confirm this issue under Windows 10 for ethereum wallet and Mist.

**Comment by psy0rz** [comment]
> Also: Should our json parser handle data from an "untrusted" source? What if someone crafts a special file that exploits a weakness in our json handling code?

Also no reply to this yet? I'm very worried that a cryptowallet makes REST calls to a 3rd party service. It basically means you have to trust all the libraries that are involved in getting and parsing json via https get request. (e.g. a lot)
